### DISCUSSION TITLE
demo里，侧边栏在有的页面显示，有的页面不显示，哪里控制的呢

### DISCUSSION THREAD

**Discussion by scil** [vuepress-theme-hope/vuepress-theme-hope#1487] [Q&A] [answered]
譬如 README.md 和 home.md，就没有显示侧边栏。有变量控制吗？

**Comment by Mister-Hope** [comment, marked as answer]
```
home: true
```

默认不启用侧边栏，这是v1默认主题，v1主题，v2默认主题，v2主题一直保持一致的逻辑。

其余的内容请看文档：

https://vuepress-theme-hope.github.io/v2/zh/guide/layout/sidebar.html

**Reply by scil** [reply]
谢谢。回复好快！赶紧 star !
